ArduPlane/fence.cpp

```diff
@@ -145,10 +139,8 @@ void Plane::fence_check()
                 }
             }
 
-            if (fence.get_action() != AC_FENCE_ACTION_RTL_AND_LAND) {
```

Comment:
This check needed a `AC_FENCE_ACTION_AUTOLAND_OR_RTL` as well. If that is added the guided case becomes completely separate.